src/tilemaps/components/CullTiles.js

```diff
@@ -45,103 +45,243 @@ var CullTiles = function (layer, camera, outputArray, renderOrder)
     var drawTop = 0;
     var drawBottom = mapHeight;
 
+    // we define the isometric culling function as a dummy early on for it to make sense in scope 
+    var inIsoBounds = function () { return true; };
```

Comment:
It would be better to define this dummy function elsewhere, instead of inside the function as part of the variable declaration. This will avoid the function definition allocating memory every time `CullTiles` is called.